modules/cms-kit/src/Volo.CmsKit.Admin.Application/Volo/CmsKit/Admin/Blogs/BlogPostAdminAppService.cs

```diff
@@ -83,15 +84,28 @@ public virtual async Task<BlogPostDto> GetAsync(Guid id)
         }
 
         [Authorize(CmsKitAdminPermissions.BlogPosts.Default)]
-        public virtual async Task<PagedResultDto<BlogPostDto>> GetListAsync(BlogPostGetListInput input)
+        public virtual async Task<PagedResultDto<BlogPostListDto>> GetListAsync(BlogPostGetListInput input)
         {
+            var blogs = (await BlogRepository.GetListAsync()).ToDictionary(x => x.Id);
+            
             var blogPosts = await BlogPostRepository.GetListAsync(input.Filter, input.BlogId, input.MaxResultCount, input.SkipCount, input.Sorting);
 
             var count = await BlogPostRepository.GetCountAsync(input.Filter);
 
-            var dtoList = ObjectMapper.Map<List<BlogPost>, List<BlogPostDto>>(blogPosts);
+            var dtoList = blogPosts.Select(x => new BlogPostListDto {
+                Id = x.Id,
+                BlogId = x.BlogId,
+                BlogName = blogs[x.BlogId].Name,
+                Title = x.Title,
+                Slug = x.Slug,
+                ShortDescription = x.ShortDescription,
+                Content = x.Content,
+                CoverImageMediaId = x.CoverImageMediaId,
+                CreationTime = x.CreationTime,
+                LastModificationTime = x.LastModificationTime
+            }).ToList();
```

Comment:
What about to use mapping here?
```suggestion
            var dtoList = blogPosts.Select(x =>
            {
                var dto = ObjectMapper.Map<BlogPost, BlogPostDto>();
                dto.BlogName = blogs[x.BlogId].Name;
                return dto;
            }}).ToList();
```

Please don't commit this suggestion directly, it might has compile error, I've just written in GitHub UI.